### PR TITLE
Rayfile zemax

### PR DESCRIPTION
**Pull request by SandAuriol**
I have added a test to check that the converted ray file works in Zemax.

### CODE REVIEW

**Comment by pluAtAnsys** [comment]
@StefanThoene @SandAuriol please check the fix solution.

File: ansys_optical_automation/zemax_process/base.py [hunk 1]
```diff
@@ -0,0 +1,160 @@
...
+        Returns
+        -------
+
+
+        """
+        if self.the_system is None:
+            raise BaseZOS.SystemNotPresentException("Unable to acquire Primary system")
+        self.the_system.LoadFile(file_path, save_if_needed)
+
+    def close_file(self, save):
```
Comment (pluAtAnsys) [inline ansys_optical_automation/zemax_process/base.py:100]: please add docstring to function

File: tests/workflows/run_test_08_rayfile.py [hunk 2]
```diff
@@ -106,7 +117,90 @@ def check_speos_sim(rayfile_path):
     return result
 
 
+def check_zos_sim(rayfile_path):
```
Comment (pluAtAnsys) [inline tests/workflows/run_test_08_rayfile.py:120]: I do not see such function has been used in test?

File: ansys_optical_automation/zemax_process/base.py [hunk 3]
```diff
@@ -0,0 +1,160 @@
...
+    class ConnectionException(Exception):
+        pass
+
+    class InitializationException(Exception):
+        pass
+
+    class SystemNotPresentException(Exception):
+        pass
+
+    def __init__(self, path=None):
```
Comment (pluAtAnsys) [inline ansys_optical_automation/zemax_process/base.py:23]: does ZOS provides a way to specify using a certain version?

**Review by StefanThoene — APPROVED** [review]

**Review by pluAtAnsys — APPROVED** [review]

**Comment by pluAtAnsys** [comment]
If there is no other comments, I will merge this PR later today :) @SandAuriol @mcheng2022 @StefanThoene

**Pull request merged**